packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -94,7 +94,7 @@ if (!ShadowRoot.prototype.createElement) {
       // Flush whenDefined callbacks
       const resolver = this._definedResolvers.get(tagName);
       if (resolver) {
-        resolver();
+        resolver(elementClass);
```

Comment:
I think ideally we should clear the `_definedResolvers` and `_definedPromises` maps. After resolving, anyone who newly calls `whenDefined()` can get a fresh Promise.

Reply:
Done